Automattic/wp-calypso
# Woop landing page: enable woop flag in production

#### Changes proposed in this Pull Request

* Enable woop flag in production.
* Only redirect the landing page if the flag isn't enabled.

#### Testing instructions



* Go to /woocommerce-installation/{site-id} for Simple sites with Free, Personal, Premium, and Business plans. 
* Click the CTA on the landing page and verify that it goes to /start/woocommerce-install.
* Regression: Sites with Woo already installed get redirected to wc-admin.



Related to #58385
